blog/2024-03-14-gsoc.md

```diff
@@ -0,0 +1,14 @@
+---
+author: Antonin Delpeuch
+title: "OpenRefine is participating in Google Summer of Code"
+---
+
+OpenRefine will be offering internships as part of the [Google Summer of Code](https://summerofcode.withgoogle.com/) this year.
+<!--truncate-->
+For prospective interns, this is an opportunity to get familiar with open source development with a paid and fully remote internship.
+We have proposed [a selection of possible internship topics](https://github.com/OpenRefine/OpenRefine/wiki/GSoC-Outreachy-2024-Ideas), but applicants are also welcome to propose other areas to work on.
+We would also welcome seasoned OpenRefine contributors to propose internship subjects and mentor interns.
+For users, this is a good opportunity to request small changes to the tool, as applicants are looking for good first issues to tackle during the application phase. This week's community meetup was
```

Comment:
Is the line break at the end of this line intentional?

Reply:
good catch, removing